cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/templates/etl/batch/ETLBatchTemplate.java

```diff
@@ -106,11 +117,78 @@ public void configureAdapter(String adapterName, ETLBatchConfig etlBatchConfig,
                                                    cronEntry));
   }
 
+  private void instantiateStages(ETLStage source, ETLStage sink, List<ETLStage> transformList)
+    throws InstantiationException, IllegalAccessException {
+    try {
+      String sourceClassName = sourceClassMap.get(source.getName());
+      String sinkClassName = sinkClassMap.get(sink.getName());
+      batchSource = (BatchSource) Class.forName(sourceClassName).newInstance();
+      batchSink = (BatchSink) Class.forName(sinkClassName).newInstance();
+
+      for (ETLStage etlStage : transformList) {
+        String transformName = transformClassMap.get(etlStage.getName());
+        Transform transformObj = (Transform) Class.forName(transformName).newInstance();
+        transforms.add(transformObj);
+      }
+    } catch (ClassNotFoundException e) {
+      throw new IllegalArgumentException(String.format("Unable to load class. Check stage names. %s", e));
```

Comment:
Don't need to %s and string format. Can just have the string message as first arg and the exception as the second.